### PR TITLE
feat(user): hide the non-message user in the sidebar AP-116

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

Hidden the users has no messages in the sidebar

**Which issue(s) this PR fixes** 🔨

AP-116

### CODE REVIEW

File: components/views/user/User.vue [hunk 1]
```diff
@@ -79,6 +79,26 @@ export default Vue.extend({
...
+    existMessage(textileObj: Conversation) {
+      const currentUserInfo = textileObj[this.user.address]
+      if (
+        currentUserInfo?.lastUpdate <= 0 &&
+        this.user.account.from === this.user.address
+      ) {
+        this.$data.existConversation = false
+      } else {
+        this.$data.existConversation = true
+      }
```
Comment (molimauro) [inline components/views/user/User.vue:91]: here we can substitute the if-else with a guard ;)

Comment (davidchan1219) [reply components/views/user/User.vue]: @molimauro I've just simplify the code

File: components/views/user/User.vue [hunk 2]
```diff
@@ -79,6 +79,23 @@ export default Vue.extend({
       })
       this.$store.commit('ui/setUserProfile', this.user)
     },
+    existMessage(textileObj: Conversation) {
+      const currentUserInfo = textileObj[this.user.address]
+      this.$data.existConversation =
+        currentUserInfo?.lastUpdate <= 0 &&
+        this.user.account.from === this.user.address
+          ? false
+          : true
```
Comment (molimauro) [inline components/views/user/User.vue:88]: at this point go this way: 
```
this.$data.existConversation = !(currentUserInfo?.lastUpdate <= 0 &&
this.user.account.from === this.user.address)
```

**Comment by stavares843** [comment]
added changes requested based on mauro's review

File: components/views/user/User.vue (hunk 2 above)
Comment (davidchan1219) [reply components/views/user/User.vue:88]: @molimauro Thanks for your comment

Comment (molimauro) [reply components/views/user/User.vue]: np!! :)

**Comment by davidchan1219** [comment]
@stavares843 fixed

**Comment by phillsatellite** [comment]
Tested: Seems like if you add a friend and the friend accepts it, they will appear in the Sidebar

https://user-images.githubusercontent.com/93608357/151889121-3da16e99-81d2-4649-9228-61ace533b3b6.mov

@davidchan1219

**Pull request closed without merging**